<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rtsp_Dialog_class</class>
 <widget class="QDialog" name="Rtsp_Dialog_class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(153, 193, 241);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QRadioButton" name="rb_activate_S1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>activate S1</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_activate_S0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>activate S0</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_activate_S2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>activate S2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_activate_S3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>activate S3</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QTableWidget" name="process_table">
     <property name="rowCount">
      <number>5</number>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pB_Start_Rtsp">
     <property name="text">
      <string>Start Rtsp</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pB_Exit_Rtsp">
     <property name="text">
      <string>Exit Rtsp</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
